src/main/app/src/interfaces.ts

```diff
@@ -1,10 +1,35 @@
 export interface ILandscape {
-    name: string;
-    identifier: string;
-    description?: string;
-    contact?: string;
-    teams: string[];
-    groups?: string[];
-    items?: string[];
-    lastUpdate?: string;
+  name: string;
+  identifier: string;
+  description?: string;
+  teams?: object[]; // TODO: Create interface
+  items?: IItem[];
+  groups?: object[]; // TODO: Create interface
+  lastUpdate?: string;
+  overallState?: string;
+}
+
+export interface IItem {
+  color?: string;
+  contact?: string;
+  description?: string;
+  fill?: string;
```

Comment:
We should think about removing the attributes width and height from the API. These are leftovers from times when to hex map was drawn in the frontend.

Reply:
see #137